Source/DafnyPipeline.Test/DafnyPipeline.Test.csproj

```diff
@@ -43,5 +43,8 @@
 	  <Content Remove="_plugins/**" />
 	  <EmbeddedResource Include="_plugins/**" />
     <None Remove="_plugins\**" />
+    <None Update="expectedProverLog.txt">
```

Comment:
It's a tiny thing, but I might call this `expectedProverLog.smt2` or `SomeProgram.smt2`.